blog/README.md

```diff
@@ -0,0 +1,83 @@
+- Open Community Issues/PRs: Link TBD
+```diff
+- Ideally we set up a query that brings up open PRs associated to the blog
+```
+
+## Submit a Post
+
+Anyone can write a blog post and submit it for review.
+```diff
+- We have to decide here if we allow commercial blogposts or not. It would also be useful if we stablish a difference between that and vendor-specific how-to blogs
+```
+
+To submit a blog post, follow the steps below.
+
+1. [Sign the Contributor License Agreements](https://github.com/knative/community/blob/master/CONTRIBUTING.md#contributor-license-agreements) if you have not yet done so.
+1. Familiarize yourself with the Markdown format for existing blog posts in the [docs repository](https://github.com/knative/docs/tree/master/blog). Blog posts are categorized into different directories. You can explore the directories to find examples.
+1. Write your blog post in a text editor of your choice.
+1. Choose a directory in the [docs repository](https://github.com/knative/docs/tree/master/blog), and click **Create new file**.
+1. Paste your content into the editor and save it. Name the file in the following way: *[BLOG] Your proposed title* , but don’t put the date in the file name. The blog reviewers will work with you on the final file name, and the date on which the blog will be published.
+1. When you save the file, GitHub will walk you through the pull request (PR) process.
+1. A reviewer is assigned to all pull requests automatically. The reviewer checks your submission, and works with you on feedback and final details. When the pull request is approved, the blog will be scheduled for publication.
+
+### Blog Guidelines
+
+Requested content:
+
+```diff
+- Ideally, each of these links to a pull request with the specific blog template, see demo for example
+```
+
+- Knative new feature releases and project updates
+- Tutorials and demos [start a blog](https://github.com/knative/docs/pull/2511)
+- Use cases
+- Vendor specific how-to content
+- **Original content only**
+```diff
+- This section needs review
+```
+
+Unsuitable Content:
+
+```diff
+- We need to specify what this is
+```
+
+## Review Process
+
+After a blog post is submitted as a PR, it is automatically assigned to a reviewer.
+
+Each blog post requires a `lgtm` label from one copy editor, technical editor\*, and blog community manager. Once the necessary labels are in place, one of the reviewers will add an `approved` label, and schedule publication of the blog post.
+
+**NOTE:** If a blog post does not contain any technical content (for example, [Knative welcomes new TOC members](https://knative.dev/blog/2020/05/12/knative-welcomes-new-toc-members/)), the technical review can be omitted.
+
+### Embargoed Content
+
+**IMPORTANT:** The blog repository on GitHub is public. Any content that needs to remain confidential until a certain time (for example: release posts, security vulnerabilities), should be submitted through the form directly.
```

Comment:
Maybe we aren't ready for embargoed content?

Reply:
Following this, and the vote on Slack TOC channel, I removed embargoed content from the guidelines!